src/OmniSharp.DotNet/DotNetProjectSystem.cs

```diff
@@ -166,13 +166,13 @@ private void UpdateProject(string projectDirectory)
             _projectStates.Update(projectDirectory, contexts, AddProject, RemoveProject);
 
             var projectFilePath = contexts.First().ProjectFile.ProjectFilePath;
-            _fileSystemWatcher.Watch(projectFilePath, file =>
+            _fileSystemWatcher.Watch(projectFilePath, (file, verb) =>
```

Comment:
See my comment above about `type` vs. `verb`. If you go with `type`, be sure to update this as well.